meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -104,7 +30,6 @@ PROGRAMMER_EC="ite_ec"
 [ -z "$SYSTEM_MODEL" ] && error_exit "SYSTEM_MODEL not given"
 
 CMD="$1"
-CMD_CLOUD_LIST="/usr/sbin/cloud_list"
 board_config() {
```

Comment:
@macpijan looks like we are using the same binaries for ec_transition, could this function also go to dts-functions.sh as a common part? Or maybe we can get rid off ec_transition script at all? What is the logical difference between EC transition and initial deployment of Dasharo on platforms like NV4x and NS50? Is there a way to change EC only without touching BIOS? If someone wants to use Dasharo on these laptops, can he do it without changing EC?

Reply:
There are only slight difference in board config for NV4x

```
PROGRAMMER_BIOS="internal:boardmismatch=force"
PROGRAMMER_EC="ite_ec:boardmismatch=force,romsize=128K,autoload=disable"
```

Reply:
> What is the logical difference between EC transition and initial deployment of Dasharo on platforms like NV4x and NS50? 

Initial deployment deploys both BIOS and EC Dasharo firmware on machine which was previously using stock firmwares (both BIOS and EC).

Transition was meant to update Dasharo BIOS and **transition** EC firwmare fro proprietary to open Dasharo EC firmware.

> Is there a way to change EC only without touching BIOS

It is, but not recommended. We should not give such option for end-users. For end users we should always keep both firmwares in sync.

> If someone wants to use Dasharo on these laptops, can he do it without changing EC?

Not really. He could install <1.3.0 releases, but we do not recommend it, of course.

Reply:
ec_transition was moved into Dasharo update https://github.com/Dasharo/meta-dts/pull/29/commits/37b0a3f010b63361e06ce1ecb64f710a7bda33f9